components/pages/explorer/RepoTable/index.tsx

```diff
@@ -342,15 +384,15 @@ const RepoTable = (props: PageContentProps): ReactElement => {
     const setId = await saveSet(sqon);
     logEvent({
       category: 'Downloads',
-      action: 'Archvie Build',
+      action: 'Archive Build',
     });
     return await startArchiveBuildBySetId(setId);
   };
 
   const updateArchiveState = async () => {
     if (archive?.status === 'BUILDING') {
       await sleep(5000);
-      await findArchvieById(archive.id).then(setArchive);
+      await findArchiveById(archive.id).then(setArchive);
```

Comment:
All remaining changes are just improving this typo someone made in many files across the app